diff --git a/node_modules/prettier-plugin-import-sort/src/index.js b/node_modules/prettier-plugin-import-sort/src/index.js
index 1ade531..12f5938 100644
--- a/node_modules/prettier-plugin-import-sort/src/index.js
+++ b/node_modules/prettier-plugin-import-sort/src/index.js
@@ -8,7 +8,7 @@ const { parsers: typescriptParsers } = _interopDefault(
   require('prettier/parser-typescript')
 )
 const { parsers: javascriptParsers } = _interopDefault(
-  require('prettier/parser-babylon')
+  require('prettier/parser-babel')
 )
 const sortImports = _interopDefault(require('import-sort'))
 const { getConfig } = _interopDefault(require('import-sort-config'))
